Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM ubuntu:focal
```

Comment:
It should be easier to take `mcr.microsoft.com/playwright:focal` as base and install jdk and driver stuff on top of it.

Reply:
The base image contains nodejs yarn and it is installing playwright from source which will make the image consume more bandwidth, because of this reason the python Docker Image is also built from source not by `mcr.microsoft.com/playwright:focal`

Reply:
Sounds like the problem will be common for all language bindings, maybe we can build a base image with the bare minimum of deps required to launch browsers and then build an image with driver on top of it and use it for both python/java etc. I'll discuss this with other folks and come back.

Reply:
ok, we are going to split `mcr.microsoft.com/playwright:focal` into base which only includes browsers and their dependencies and move node.js specific stuff into an image derived from it. Java and Python bindings could be derived from the same base and install only language dependencies on top of it.

cc @aslushnikov

Reply:
Marking as draft until the base image is created

Reply:
PR on the way 🚀 https://github.com/microsoft/playwright/pull/4632